README.md

```
@@ -38,6 +39,19 @@ Dedicated to making your workflow more seamless. Search everything from applicat
+## 📣 Official Channels
+
+- Website: [https://flowlauncher.com](https://flowlauncher.com)
+- GitHub Organization: [https://github.com/Flow-Launcher](https://github.com/Flow-Launcher)
+- Discord: [https://discord.gg/AvgAQgh](https://discord.gg/AvgAQgh)
+- Reddit: [https://www.reddit.com/r/FlowLauncher/](https://www.reddit.com/r/FlowLauncher/)
+
+⚠️ Only trust official channels for downloads and announcements, and be careful of similar-looking domains.
+
+For installation, use the official methods listed below.
+
+<img src="https://user-images.githubusercontent.com/6903107/144858082-8b654daf-60fb-4ee6-89b2-6183b73510d1.png" width="100%">
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)